<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="10" total="18" passed="1" failed="7" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-09-27T06:48:04Z" name="Suite" finished-at="2023-09-27T06:56:40Z" duration-ms="515684">
    <groups>
    </groups>
    <test started-at="2023-09-27T06:48:04Z" name="Console Application Regression Suite" finished-at="2023-09-27T06:56:40Z" duration-ms="515684">
      <class name="com.qa.dt.test.AppWorkspaceSelectTest">
        <test-method is-config="true" signature="loginTheApplication()[pri:0, instance:com.qa.dt.test.AppWorkspaceSelectTest@5dbe30be]" started-at="2023-09-27T06:48:05Z" name="loginTheApplication" finished-at="2023-09-27T06:48:26Z" duration-ms="21948" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTheApplication -->
        <test-method signature="verifyApplicationProjectSelection()[pri:0, instance:com.qa.dt.test.AppWorkspaceSelectTest@5dbe30be]" started-at="2023-09-27T06:48:26Z" name="verifyApplicationProjectSelection" finished-at="2023-09-27T06:49:14Z" duration-ms="47419" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [digitaltwin | 2-4 Wilton Park] but found [PlatformReferenceApp | cTest]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [digitaltwin | 2-4 Wilton Park] but found [PlatformReferenceApp | cTest]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.qa.dt.base.BaseClass.comparetwoText(BaseClass.java:761)
at com.qa.dt.page.ProjectSelection.verifyAppProjectSelection(ProjectSelection.java:92)
at com.qa.dt.test.AppWorkspaceSelectTest.verifyApplicationProjectSelection(AppWorkspaceSelectTest.java:34)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyApplicationProjectSelection -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.qa.dt.test.AppWorkspaceSelectTest@5dbe30be]" started-at="2023-09-27T06:49:14Z" name="tearDown" finished-at="2023-09-27T06:49:18Z" duration-ms="3759" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="loginTheApplication()[pri:0, instance:com.qa.dt.test.AppWorkspaceSelectTest@5dbe30be]" started-at="2023-09-27T06:49:18Z" name="loginTheApplication" finished-at="2023-09-27T06:49:37Z" duration-ms="19124" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTheApplication -->
        <test-method signature="verifyChangeAppProjectSelection()[pri:0, instance:com.qa.dt.test.AppWorkspaceSelectTest@5dbe30be]" started-at="2023-09-27T06:49:37Z" name="verifyChangeAppProjectSelection" finished-at="2023-09-27T06:50:01Z" duration-ms="23964" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [digitaltwin | 2-4 Wilton Park] but found [PlatformReferenceApp | cTest]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [digitaltwin | 2-4 Wilton Park] but found [PlatformReferenceApp | cTest]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.qa.dt.base.BaseClass.comparetwoText(BaseClass.java:761)
at com.qa.dt.page.ProjectSelection.verifyAppProjectSelection(ProjectSelection.java:92)
at com.qa.dt.test.AppWorkspaceSelectTest.verifyChangeAppProjectSelection(AppWorkspaceSelectTest.java:56)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyChangeAppProjectSelection -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.qa.dt.test.AppWorkspaceSelectTest@5dbe30be]" started-at="2023-09-27T06:50:01Z" name="tearDown" finished-at="2023-09-27T06:50:06Z" duration-ms="4602" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.dt.test.AppWorkspaceSelectTest -->
      <class name="com.qa.dt.test.ItemServiceRegression">
        <test-method is-config="true" signature="precondition()[pri:0, instance:com.qa.dt.test.ItemServiceRegression@55795845]" started-at="2023-09-27T06:52:10Z" name="precondition" finished-at="2023-09-27T06:52:10Z" duration-ms="40" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method is-config="true" signature="loginTheApplication()[pri:0, instance:com.qa.dt.test.ItemServiceRegression@55795845]" started-at="2023-09-27T06:52:10Z" name="loginTheApplication" finished-at="2023-09-27T06:52:32Z" duration-ms="21510" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTheApplication -->
        <test-method signature="verifyItemServicesTable()[pri:1, instance:com.qa.dt.test.ItemServiceRegression@55795845]" started-at="2023-09-27T06:52:32Z" name="verifyItemServicesTable" finished-at="2023-09-27T06:53:14Z" duration-ms="41634" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Application] but found [PlatformReferenceApp | cTest]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Application] but found [PlatformReferenceApp | cTest]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.qa.dt.base.BaseClass.comparetwoText(BaseClass.java:761)
at com.qa.dt.page.ProjectSelection.verifyAppProjectSelection(ProjectSelection.java:92)
at com.qa.dt.page.ProjectSelection.selectApplicationAndWorkspace(ProjectSelection.java:111)
at com.qa.dt.test.ItemServiceRegression.verifyItemServicesTable(ItemServiceRegression.java:56)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyItemServicesTable -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.qa.dt.test.ItemServiceRegression@55795845]" started-at="2023-09-27T06:53:14Z" name="tearDown" finished-at="2023-09-27T06:53:19Z" duration-ms="5132" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.dt.test.ItemServiceRegression -->
      <class name="com.qa.dt.test.UsergroupRegression">
        <test-method is-config="true" signature="precondition()[pri:0, instance:com.qa.dt.test.UsergroupRegression@3d08f3f5]" started-at="2023-09-27T06:53:19Z" name="precondition" finished-at="2023-09-27T06:53:19Z" duration-ms="50" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method is-config="true" signature="loginTheApplication()[pri:0, instance:com.qa.dt.test.UsergroupRegression@3d08f3f5]" started-at="2023-09-27T06:53:19Z" name="loginTheApplication" finished-at="2023-09-27T06:53:40Z" duration-ms="21285" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTheApplication -->
        <test-method signature="verifyUserGroupValidations()[pri:1, instance:com.qa.dt.test.UsergroupRegression@3d08f3f5]" started-at="2023-09-27T06:53:40Z" name="verifyUserGroupValidations" finished-at="2023-09-27T06:54:23Z" duration-ms="43065" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at com.qa.dt.base.BaseClass.failedStep(BaseClass.java:1096)
at com.qa.dt.page.CommonReusablesPage.verifyTableHeaders(CommonReusablesPage.java:223)
at com.qa.dt.test.UsergroupRegression.verifyUserGroupValidations(UsergroupRegression.java:68)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUserGroupValidations -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.qa.dt.test.UsergroupRegression@3d08f3f5]" started-at="2023-09-27T06:54:24Z" name="tearDown" finished-at="2023-09-27T06:54:28Z" duration-ms="4098" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.dt.test.UsergroupRegression -->
      <class name="com.qa.dt.test.FileServicesRegression">
        <test-method is-config="true" signature="precondition()[pri:0, instance:com.qa.dt.test.FileServicesRegression@4fe89c24]" started-at="2023-09-27T06:51:03Z" name="precondition" finished-at="2023-09-27T06:51:04Z" duration-ms="709" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method is-config="true" signature="loginTheApplication()[pri:0, instance:com.qa.dt.test.FileServicesRegression@4fe89c24]" started-at="2023-09-27T06:51:04Z" name="loginTheApplication" finished-at="2023-09-27T06:51:24Z" duration-ms="19841" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTheApplication -->
        <test-method signature="verifyHierarchyOfFoldersDisplayed()[pri:1, instance:com.qa.dt.test.FileServicesRegression@4fe89c24]" started-at="2023-09-27T06:51:24Z" name="verifyHierarchyOfFoldersDisplayed" finished-at="2023-09-27T06:52:05Z" duration-ms="41391" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Application] but found [PlatformReferenceApp | cTest]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Application] but found [PlatformReferenceApp | cTest]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.qa.dt.base.BaseClass.comparetwoText(BaseClass.java:761)
at com.qa.dt.page.ProjectSelection.verifyAppProjectSelection(ProjectSelection.java:92)
at com.qa.dt.page.ProjectSelection.selectApplicationAndWorkspace(ProjectSelection.java:111)
at com.qa.dt.test.FileServicesRegression.verifyHierarchyOfFoldersDisplayed(FileServicesRegression.java:58)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyHierarchyOfFoldersDisplayed -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.qa.dt.test.FileServicesRegression@4fe89c24]" started-at="2023-09-27T06:52:05Z" name="tearDown" finished-at="2023-09-27T06:52:10Z" duration-ms="5017" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="loginTheApplication()[pri:0, instance:com.qa.dt.test.FileServicesRegression@4fe89c24]" started-at="2023-09-27T06:55:35Z" name="loginTheApplication" finished-at="2023-09-27T06:55:55Z" duration-ms="19701" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTheApplication -->
        <test-method signature="verifyFileVersionHistoryandBreadcrumb()[pri:2, instance:com.qa.dt.test.FileServicesRegression@4fe89c24]" started-at="2023-09-27T06:55:55Z" name="verifyFileVersionHistoryandBreadcrumb" finished-at="2023-09-27T06:56:37Z" duration-ms="41548" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Application] but found [PlatformReferenceApp | cTest]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Application] but found [PlatformReferenceApp | cTest]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.qa.dt.base.BaseClass.comparetwoText(BaseClass.java:761)
at com.qa.dt.page.ProjectSelection.verifyAppProjectSelection(ProjectSelection.java:92)
at com.qa.dt.page.ProjectSelection.selectApplicationAndWorkspace(ProjectSelection.java:111)
at com.qa.dt.test.FileServicesRegression.verifyFileVersionHistoryandBreadcrumb(FileServicesRegression.java:73)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyFileVersionHistoryandBreadcrumb -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.qa.dt.test.FileServicesRegression@4fe89c24]" started-at="2023-09-27T06:56:37Z" name="tearDown" finished-at="2023-09-27T06:56:40Z" duration-ms="3296" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.dt.test.FileServicesRegression -->
      <class name="com.qa.dt.test.AccountsDetailTest">
        <test-method is-config="true" signature="startReport()[pri:0, instance:com.qa.dt.test.AccountsDetailTest@89c10b7]" started-at="2023-09-27T06:48:04Z" name="startReport" finished-at="2023-09-27T06:48:04Z" duration-ms="671" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startReport -->
        <test-method is-config="true" signature="loginTheApplication()[pri:0, instance:com.qa.dt.test.AccountsDetailTest@89c10b7]" started-at="2023-09-27T06:50:06Z" name="loginTheApplication" finished-at="2023-09-27T06:50:23Z" duration-ms="17224" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTheApplication -->
        <test-method signature="verifyAccountsDetailPageAndLogout()[pri:1, instance:com.qa.dt.test.AccountsDetailTest@89c10b7]" started-at="2023-09-27T06:50:23Z" name="verifyAccountsDetailPageAndLogout" finished-at="2023-09-27T06:50:58Z" duration-ms="34987" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyAccountsDetailPageAndLogout -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.qa.dt.test.AccountsDetailTest@89c10b7]" started-at="2023-09-27T06:50:58Z" name="tearDown" finished-at="2023-09-27T06:51:03Z" duration-ms="4898" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="endReport()[pri:0, instance:com.qa.dt.test.AccountsDetailTest@89c10b7]" started-at="2023-09-27T06:56:40Z" name="endReport" finished-at="2023-09-27T06:56:41Z" duration-ms="589" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
      </class> <!-- com.qa.dt.test.AccountsDetailTest -->
      <class name="com.qa.dt.test.WorkspaceRegression">
        <test-method is-config="true" signature="precondition()[pri:0, instance:com.qa.dt.test.WorkspaceRegression@119f1f2a]" started-at="2023-09-27T06:54:28Z" name="precondition" finished-at="2023-09-27T06:54:28Z" duration-ms="29" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- precondition -->
        <test-method is-config="true" signature="loginTheApplication()[pri:0, instance:com.qa.dt.test.WorkspaceRegression@119f1f2a]" started-at="2023-09-27T06:54:28Z" name="loginTheApplication" finished-at="2023-09-27T06:54:48Z" duration-ms="20713" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTheApplication -->
        <test-method signature="verifyWorkspaceTableUiValidations()[pri:1, instance:com.qa.dt.test.WorkspaceRegression@119f1f2a]" started-at="2023-09-27T06:54:48Z" name="verifyWorkspaceTableUiValidations" finished-at="2023-09-27T06:55:31Z" duration-ms="42795" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at com.qa.dt.base.BaseClass.failedStep(BaseClass.java:1096)
at com.qa.dt.page.CommonReusablesPage.verifyTableHeaders(CommonReusablesPage.java:223)
at com.qa.dt.test.WorkspaceRegression.verifyWorkspaceTableUiValidations(WorkspaceRegression.java:66)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyWorkspaceTableUiValidations -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.qa.dt.test.WorkspaceRegression@119f1f2a]" started-at="2023-09-27T06:55:32Z" name="tearDown" finished-at="2023-09-27T06:55:35Z" duration-ms="3879" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.dt.test.WorkspaceRegression -->
    </test> <!-- Console Application Regression Suite -->
  </suite> <!-- Suite -->
</testng-results>
